src/modules/backupRestore/backupRestoreEntities/entities/assets.ts

```diff
@@ -69,9 +69,14 @@ const saveAsset = async (
   archive: archiver.Archiver,
   logOptions: LogOptions,
   asset: AssetContracts.IAssetModelContract,
+  secureAssetDeliveryKey: string | undefined,
 ) => {
   logInfo(logOptions, "verbose", `Exporting: file ${chalk.yellow(asset.file_name)}.`);
-  const file = await fetch(asset.url).then(res => res.blob()).then(res => res.stream());
+  const defaultOptions: RequestInit = {
+    headers: secureAssetDeliveryKey ? { Authorization: `Bearer ${secureAssetDeliveryKey}` } : undefined,
+  };
+  const file = await fetch(asset.url, defaultOptions)
```

Comment:
could you please update the URL with the query parameter `?q=100`? You could use something like string formatting to add it (\`${asset.url}?q=100\`) The images would have better quality then :)